wiki/Disambiguation/Snapping/en.md

```diff
@@ -0,0 +1,13 @@
+# Snapping (disambiguation)
+
+**Snapping** can have multiple meanings:
+
+## Mapping
+
+- [Beat snapping](/wiki/Beatmapping/Snapping), a hit object's position within a single musical beat, which is split automatically using [beat snap divisor](/wiki/Client/Beatmap_editor/Beat_Snap_Divisor)
+- [Grid snapping](/wiki/Grid_snapping), a hit object's position on the playfield
+
+## Gameplay
+
+- [Dash snapping](/wiki/Gameplay/Dash_snapping), an [osu!catch](/wiki/Game_mode/osu!catch) term that describes the movement required from a player to catch the next fruit
+- An [osu!](/wiki/Game_mode/osu!) play style characterized by rapid, uneven movements, when the cursor lands on hit objects quickly instead of moving uniformly between them<!-- TODO: this would be a good addition to /wiki/Play_style -->
```

Comment:
```suggestion
- An [osu!](/wiki/Game_mode/osu!) play style characterised by rapid, uneven movements, when the cursor lands on hit objects quickly instead of moving uniformly between them<!-- TODO: this would be a good addition to /wiki/Play_style -->
```